infra/ansible/roles/repository/tasks/main.yml

```diff
@@ -18,3 +18,11 @@
     version: "{{git_reference}}"
   become: yes
   become_user: git
+
+- name: Checkout Plausible Analytics self-hosting repository
+  git:
+    repo: 'https://github.com/tournesol-app/plausible-self-hosting'
+    dest: /home/git/plausible_hosting
+    version: tournesol-deployment
```

Comment:
regarding the remaining tasks:
> define the Plausible Analytics git repository update policy
 define the Plausible Analytics container update policy

maybe a git **tag** could be used here, to make any future update more explicit

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1340/commits/fd0b65a8bc4bcf8534a92c90f71c51fb3ad2e61d